src/main/java/net/neoforged/neoforge/common/data/internal/NeoForgeBlockTagsProvider.java

```diff
@@ -80,7 +80,9 @@ public void addTags(HolderLookup.Provider p_256380_) {
         tag(Tags.Blocks.SKULLS).add(Blocks.SKELETON_SKULL, Blocks.SKELETON_WALL_SKULL, Blocks.WITHER_SKELETON_SKULL, Blocks.WITHER_SKELETON_WALL_SKULL, Blocks.PLAYER_HEAD, Blocks.PLAYER_WALL_HEAD, Blocks.ZOMBIE_HEAD, Blocks.ZOMBIE_WALL_HEAD, Blocks.CREEPER_HEAD, Blocks.CREEPER_WALL_HEAD, Blocks.PIGLIN_HEAD, Blocks.PIGLIN_WALL_HEAD, Blocks.DRAGON_HEAD, Blocks.DRAGON_WALL_HEAD);
         tag(Tags.Blocks.HIDDEN_FROM_RECIPE_VIEWERS);
         tag(Tags.Blocks.NETHERRACKS).add(Blocks.NETHERRACK);
-        tag(Tags.Blocks.OBSIDIANS).add(Blocks.OBSIDIAN).add(Blocks.CRYING_OBSIDIAN);
+        tag(Tags.Blocks.OBSIDIANS_NORMAL).add(Blocks.OBSIDIAN).add(Blocks.CRYING_OBSIDIAN);
```

Comment:
Why are you adding both normal and crying obsidian to both tags?

Reply:
simple, typo